### PR TITLE
Updates to help materials

### PR DESCRIPTION
**Pull request by jenniferward**

### CODE REVIEW

File: default/en/help_faqs.md [hunk 1]
```diff
@@ -1,41 +1,41 @@
...
+Yes, you are welcome to do so. A training version of Muscat is available for exactly these purposes at [https://muscat-training.rism.info](https://muscat-training.rism.info/). All Muscat users can login there with their personal credentials. There are also 99 training accounts ("training01@rism.info" to "training99@rism.info") available for individual users. Any of these training accounts can be used. Contact the RISM Editorial Center for the current password. Anything on the training version can be added or edited. It is synchronized with the current Muscat data on a regular basis, so practice records on the training server are be deleted after these updates. New records or changes on the training server are not transferred to the main Muscat program.
```
Comment (ahankinson) [inline default/en/help_faqs.md:28]: "Synchronized" implies two-way transfer... 

```suggestion
Yes, you are welcome to do so. A training version of Muscat is available for exactly these purposes at [https://muscat-training.rism.info](https://muscat-training.rism.info/). All Muscat users can login there with their personal credentials. There are also 99 training accounts ("training01@rism.info" to "training99@rism.info") available for individual users. Any of these training accounts can be used. Contact the RISM Editorial Center for the current password. Anything on the training version can be added or edited. It is updated with the current Muscat data on a regular basis, so practice records on the training server are be deleted after these updates. New records or changes on the training server are not transferred to the main Muscat program.
```

File: default/en/introduction.md [hunk 2]
```diff
@@ -17,15 +17,15 @@ The middle sections (5-9) contain the guidelines for RISM's authority files. The
...
-Korean: JeongYoun Chang 
-Polish: Marcelina Chojecka, Katarzyna Diehl
-Portuguese: André Guerra Cotta (†), Diósnio Machado Neto, Beatriz Magalhães Castro, Silvia Sequeira, Marcos Virmond
-Spanish: Rodrigo Balaguer, Antonio Ezquerro, John Lazos, Lucas Reccitelli, Oliver Schöner, Luciana Giron Sheridan
+English: [Jennifer Ward](mailto:jennifer.ward@rism.info)  
+German: [Martina Falletta](mailto:martina.falletta@rism.info)  
+Should anything be unclear, do not hesitate to contact us.  
+
+We would also like to extend our gratitude to our translators, past and present, whose efforts make Muscat more easily accessible to our colleagues worldwide.  
+Catalan: Ferran Jorba
```
Comment (ahankinson) [inline default/en/introduction.md:25]: This section should probably be formatted as a list?

**Review by ahankinson — COMMENTED** [review]
just a few suggestions

File: default/en/introduction.md (hunk 2 above)
Comment (jenniferward) [reply default/en/introduction.md:25]: Oh I thought it was, but I forgot the dashes. Will do that on the next pass. 
All suggestions gratefully accepted!

**Pull request merged**